flipt-evaluation/src/models/source.rs

```diff
@@ -46,7 +49,8 @@ pub struct Flag {
 pub struct Variant {
     pub id: String,
     pub key: String,
-    pub attachment: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub attachment: Option<String>,
```

Comment:
now that we are 'exposing' this data publicly we need to be careful about optionals as this was previously writing the string "null" like {"attachment": "null"} which is not what we want